### PR TITLE
add accounts_deleteAccount method

### PR DESCRIPTION
**Pull request by yenda**
add deleteAccount method for accounts as well as created and updated timestamps so they can be returned ordered to status-react

### CODE REVIEW

**Comment by rasom** [comment]
@yenda so we don't remove account's database, keys, only entry from multiaccs database? is that enough?

**Comment by yenda** [comment]
@rasom it's to delete an account not a multiaccount

File: appdatabase/migrations/sql/0001_app.up.sql [hunk 1]
```diff
@@ -12,7 +12,9 @@ storage TEXT,
 pubkey BLOB,
 path TEXT,
 name TEXT,
-color TEXT
+color TEXT,
+created_at DATETIME,
```
Comment (adambabik) [inline appdatabase/migrations/sql/0001_app.up.sql]:
```suggestion
created_at DATETIME NOT NULL,
```

File: appdatabase/migrations/sql/0001_app.up.sql [hunk 2]
```diff
@@ -12,7 +12,9 @@ storage TEXT,
...
+updated_at DATETIME
```
Comment (adambabik) [inline appdatabase/migrations/sql/0001_app.up.sql]:
```suggestion
updated_at DATETIME NOT NULL
```

File: appdatabase/migrations/sql/0001_app.up.sql (hunk 1 above)
Comment (adambabik) [reply appdatabase/migrations/sql/0001_app.up.sql]: This column can benefit from having an index.

File: services/accounts/accounts.go [hunk 3]
```diff
@@ -29,3 +30,12 @@ func (api *API) SaveAccounts(ctx context.Context, accounts []accounts.Account) e
 func (api *API) GetAccounts(ctx context.Context) ([]accounts.Account, error) {
 	return api.db.GetAccounts()
 }
+
+func (api *API) DeleteAccount(ctx context.Context, address common.Address) error {
+	err := api.db.DeleteAccount(address)
+	if err != nil {
+		return err
+	}
+	api.feed.Send(address)
```
Comment (adambabik) [inline services/accounts/accounts.go]: Do we need this? `api.feed` takes various arguments, not sure what it is useful for.

File: appdatabase/migrations/sql/0001_app.up.sql (hunk 1 above)
Comment (andremedeiros) [reply appdatabase/migrations/sql/0001_app.up.sql]: I can't imagine the volume would make much of a difference, but I'm on it.

**Review by andremedeiros — APPROVED** [review]

**Review by adambabik — APPROVED** [review]

**Pull request merged**